### DISCUSSION TITLE
MacOS Host and Ubuntu Guest - use command to copy/paste within Ubuntu

### DISCUSSION THREAD

**Discussion by fiore42** [utmapp/UTM#4756] [General]
(this was tested with Ubuntu 22.04 desktop for ARM guest and MacBook Air M2 host)

I couldn't easily figure out how to remap Command and Control and now I think I solved it, so sharing it here for others.

In Ubuntu add "English (Macintosh)" keyboard layout following [this guide](https://askubuntu.com/questions/1302185/missing-apple-keyboard-layout-english-international-macintosh) and search _"I feel stupid for not figuring this out myself"_

And then swap Control and Command keys following [this guide](https://askubuntu.com/questions/137172/how-to-remap-super-left-key-to-control-key) and search _"If you want to do with with a GUI:"_

Hope this helps!

**Comment by Crown0815** [comment]
Direct link to the _"I feel stupid for not figuring this out myself"_ answer: https://askubuntu.com/a/1307367/1782858
Direct link to the _"If you want to do with with a GUI:"_  answer: https://askubuntu.com/a/1160086/1782858
